flycheck.el

```diff
@@ -5351,6 +5353,29 @@ See URL `http://phpmd.org/' for more information about phpmd."
                       errors)))))))))
        (nreverse errors)))))
 
+(defun flycheck-parse-tslint (output checker buffer)
+  "Parse TSLint errors from JSON OUTPUT.
+
+CHECKER and BUFFER denoted the CHECKER that returned OUTPUT and
+the BUFFER that was checked respectively.
+
+See URL `https://palantir.github.io/tslint/' for more information
+about TSLint."
+  (let ((json-array-type 'list))
+    (let ((errors)
+          (tslint-json-output (json-read-from-string output)))
+      (dolist (emessage tslint-json-output)
+        (let-alist emessage
+          (push (flycheck-error-new-at
+                 (+ 1 .startPosition.line)
+                 (+ 1 .startPosition.character)
+                 'warning .failure
```

Comment:
Is there no level or severity in the output of `tslint`? Is really everything that tslint reports a warning?